suite/tests/client-interface/features.dll.c

```diff
@@ -117,7 +117,7 @@ static const struct feature_strings features[] = {
     { "sha512", FEATURE_SHA512 }, { "atomics", FEATURE_LSE }, { "bf16", FEATURE_BF16 },
     { "jscvt", FEATURE_JSCVT },   { "lrcpc", FEATURE_LRCPC }, { "sm3", FEATURE_SM3 },
     { "sm4", FEATURE_SM4 },       { "i8mm", FEATURE_I8MM },   { "rng", FEATURE_RNG },
-    { "fphp", FEATURE_FP16 }
+    { "fphp", FEATURE_FP16 },     { "mte", FEATURE_MTE },
```

Comment:
Does `mte` in `/proc/cpuinfo`'s `Features` string also cover MTE2?

Reply:
This should have been `FEATURE_MTE2` actually. The "mte" feature string indicates `FEAT_MTE2` support. Linux doesn't appear to report `FEAT_MTE` separately.